File: backends/kotlin-ktor/src/main/kotlin/com/authorization/domain/invitation/ValueObjects.kt
/*
 * 招待ドメインの値オブジェクトモジュール。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
package com.authorization.domain.invitation

/**
 * 招待情報の値オブジェクトです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
data class Vo(
    val token:      String,
    val role:       Int,
    val url:        String,
    val displayUrl: String,
)

File: backends/rust-axum/src/domain/invitation/value_objects.rs
//! 招待ドメイン 値オブジェクトモジュール。
//!
//! # Author
//! Satoshi Nagashiba <satoshi.nagashiba@gmail.com>

/// 招待トークン取得・発行結果の VO。
pub struct Vo {
    pub token:       String,
    pub role:        u8,
    pub url:         String,
    pub display_url: String,
}
